charts/opentelemetry-operator/values.yaml

```diff
@@ -160,7 +159,8 @@ manager:
 
   ## Container specific securityContext
   ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-container
-  securityContext: {}
+  securityContext:
+    {}
```

Comment:
I assume this is a linter, but I'm not sure if it is equivalent to `securityContext: {}`.  Can we convert these back?

Reply:
yeah this is a linter and is equivalent, i can swap it back...

Reply:
i think its doing it because the comment is indented down. I'll fix that too

Reply:
similar to this above:
```
    minAllowed: {}
    # cpu: 200m
    # memory: 100Mi
```